_posts/2021-04-21-the-definitive-guide.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title:  "Trino: The Definitive Guide"
+author: Matt Fuller, Manfred Moser and Martin Traverso
+excerpt_separator: <!--more-->
+---
+
+Just over a year ago we [announced the availability of the first book about
+Trino]({{ site.url }}{% post_url 2020-04-11-the-definitive-guide %}) - our
+definitive guide. Back then the project was still called Presto, and the rename
+with the end of 2020 was a good reason for us to give the book a refresh.
+
+Today, we are happy to announce that a new edition now titled __Trino: The
+Definitive Guide__ is available.
+
+> ## [Get a free copy of Trino: The Definitive Guide](https://www.starburst.io/info/oreilly-trino-guide/) from [Starburst](https://www.starburst.io) now!
+
+<!--more-->
+
+<img src="/assets/ttdg-cover.png" align="right" style="float: right; margin-left: 20px; margin-bottom: 20px; width: 100%; max-width: 350px;">
+
+The new edition of the book from O'Reilly is again available in digital formats
+as well as copies. You can find more information about the book on [our
+permanent page about it](/trino-the-definitive-guide.html).
+
+The book is now updated to Trino release 354 for all filenames, installation
+methods, command, names and properties. We addressed all problems found by our
+readers and reported to us as well.
+
+Our major supporter, [Starburst](https://www.starburst.io), allowed us to work
+on the book and bring it across the finish line again. And not only is the book
```

Comment:
Remove "And not only is the book finished now", and just say "You can get a free digital copy..."

Reply:
Done